doc/manual/source/installation.rst

```diff
@@ -4,10 +4,19 @@ Installation
 System Requirements
 -------------------
 
-Routinator has minimal system requirements. When choosing a system, make sure
-you have 1GB of available memory and 4GB of disk space for the application.
-This will give you ample margin for the RPKI repositories to grow over time,
-as adoption increases. A powerful CPU is not required.
+Routinator has minimal system requirements. When choosing a system, a
+powerful CPU is not required. Make sure you have 1GB of available memory and
+4GB of disk space for the application. 
+
+Please keep in mind that the RPKI consists of a very large number of small
+files. As a result, Routinator will use a large number of inodes. You should
+accommodate for at least 500,000 inodes, but one million will provide more
+breathing room. This will give you ample margin for the RPKI repositories to
+grow over time, as adoption increases. 
+
+.. Note:: A "No space left on device" error may be caused by running out of 
+          inodes instead of disk space. The ``du --inode`` command will list 
```

Comment:
You might want to also mention `df -i` to show the amount of inodes available, used, and free.

Reply:
Actually, I think `df -i` is the more helpful command in this case. `du --inode` tells you how many inodes Routinator is currently using (if you run it in the right directory).